simapp/app.go

```diff
@@ -591,6 +608,11 @@ func (app *SimApp) TxConfig() client.TxConfig {
 	return app.txConfig
 }
 
+// DefaultGenesis returns a default genesis from the registered AppModuleBasic's.
+func (a *SimApp) DefaultGenesis() map[string]json.RawMessage {
```

Comment:
Why do we need this?

Reply:
Because we replace `NewDefaultGenesisState` by this. We should limit the usage of ModuleBasics in general.